app/src/components/widgets/charts/launchesDurationChart/tooltip.jsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { duration } from 'moment';
+import classNames from 'classnames/bind';
 
 import { isValueInterrupted } from './chartUtils';
 
+import styles from './launchesDurationChart.scss';
```

Comment:
Please make a new file `launchDurationTooltip.scss` and put corresponding styles there.
Also, this file should be renamed according to the component's name.
```
launchesDurationChart
  |-launchDurationTooltip
    |-launchDurationTooltip.jsx
    |-launchDurationTooltip.scss
    |-index.js
```

Reply:
Done